cmd/daemon/flux-notification-controller/api.go

```diff
@@ -0,0 +1,35 @@
+package flux_notification_controller
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/lunarway/release-manager/internal/log"
+)
+
+type Event struct {
+	InvolvedObject struct {
+		Kind            string `json:"kind"`
+		Namespace       string `json:"namespace"`
+		Name            string `json:"name"`
+		UID             string `json:"uid"`
+		APIVersion      string `json:"apiVersion"`
+		ResourceVersion string `json:"resourceVersion"`
+	} `json:"involvedObject"`
+	Severity            string    `json:"severity"`
+	Timestamp           time.Time `json:"timestamp"`
+	Message             string    `json:"message"`
+	Reason              string    `json:"reason"`
+	ReportingController string    `json:"reportingController"`
+	ReportingInstance   string    `json:"reportingInstance"`
+}
+
+func HandleEventFromFlux2(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	body, _ := ioutil.ReadAll(r.Body) //TODO: log something
+	var event Event
+	_ = json.Unmarshal(body, &event)
+	log.Infof("Received alert from flux2-notification-controller: %s with msg: %s", event.InvolvedObject.Name, event.Message)
```

Comment:
Should we get the logging in place here up front to avoid more turn around if something works unexpectedly?
The most common case of errors here would be that our `Event` types doesn't match what we receive.

Consider this change. I add logging and an error response to let the caller know something went wrong and we get some context to it as well. WDYT?

```suggestion
	body, err := ioutil.ReadAll(r.Body) //TODO: log something
	if err != nil {
		log.Errorf("Failed to unmarshal alert from flux2-notification-controller: %v", err)
		http.Error(w, "unknown error", http. StatusInternalServerError)
		return
	}
	
	var event Event
	err = json.Unmarshal(body, &event)
	if err != nil {
		log.Errorf("Failed to unmarshal alert from flux2-notification-controller: %v", err)
		http.Error(w, "unknown error", http. StatusInternalServerError)
		return
	}
	log.Infof("Received alert from flux2-notification-controller: %s with msg: %s", event.InvolvedObject.Name, event.Message)
```

We _could_ consider adding a payload log to ease debugging as well in the unmarshal case to have the "real" payload logged out. Something like `log.Infof("Alert payload: %s", body)`